s2a/src/main/java/io/grpc/s2a/channel/S2AHandshakerServiceChannel.java

```diff
@@ -140,23 +130,21 @@ public String toString() {
   }
 
   /**
-   * Manages a channel using a {@link ManagedChannel} instance that belong to the {@code
-   * EventLoopGroup} thread pool.
+   * Manages a channel using a {@link ManagedChannel} instance.
    */
   @VisibleForTesting
-  static class EventLoopHoldingChannel extends Channel {
+  static class HandshakerServiceChannel extends Channel {
```

Comment:
ChannelResource doesn't need this class any longer. Delete it. (If you want to do it in a follow-up, that's fine.)

Reply:
Done in https://github.com/grpc/grpc-java/pull/11584/commits/a15421b77fe995bd8a58353959160eb8e74a3cb5